packages/foundation-models/src/azure-openai/azure-openai-chat-client.test.ts

```diff
@@ -4,16 +4,16 @@ import {
   mockInference,
   parseMockResponse
 } from '../../../../test-util/mock-http.js';
-import {
-  AzureOpenAiChatCompletionOutput,
-  AzureOpenAiChatMessage
-} from './azure-openai-types.js';
 import { AzureOpenAiChatClient } from './azure-openai-chat-client.js';
+import type {
+  AzureOpenAiChatCompletionRequestMessageUser,
+  AzureOpenAiCreateChatCompletionResponse
+} from './client/inference/schema/index.js';
 
 describe('Azure OpenAI chat client', () => {
   const chatCompletionEndpoint = {
     url: 'inference/deployments/1234/chat/completions',
-    apiVersion: '2024-02-01'
+    apiVersion: '2024-06-01'
```

Comment:
[q] Can we also import this since you defined already a const? I feel it won't affect the test.

Reply:
good point. I'll change this